tests/asconfig/flags/nested/assembly/index.ts

```diff
@@ -0,0 +1 @@
+
```

Comment:
```suggestion
```

Reply:
Should we just remove this test?

Reply:
The suggestion is just removing the blank line, not the file. Or do you think we don't need the test at all?

Reply:
The flags test appears to be empty completely.

Reply:
The reasonably complicated test appears to be working, so I am going to remove that folder and test.

Reply:
Figured as much, but I guess you know better than me why that is? Would assume that `asc` requires an entry file to exists to perform an invocation test?